collections/sentinel-1-grd.yaml

```diff
@@ -26,11 +26,17 @@ Tags:
   - systematic
   - satellite imagery
   - SAR
+  - radar
+  - backscattering
+  - polarization
+  - Copernicus
   - maritime monitoring
   - land monitoring
   - disaster response
   - open data
   - race challenges
+  - Copernicus
+  - sentinel
```

Comment:
Copernicus is added twice.

Should "sentinel" start with an upper letter?

Reply:
Removed one Copernicus.

> Should "sentinel" start with an upper letter? 

The styling now is that all tags are starting with lower letter besides those that are acronyms like NASA, SAR... This might not be the best system but think it's okay for now as it's consistent among all files.

Reply:
> The styling now is that all tags are starting with lower letter besides those that are acronyms

That's fine, too. Just wondered if there is some convention in place already. The updated tags look ok now.